File: Completed-Algorithms/LeetCode1_Two_Sum/Two_Sum.js
const twoSum = (nums, target) => {
    let start = 0
    while (true) {
    for (let i = start + 1; i < nums.length; i++) {
        if (nums[start] + nums[i] === target ) {
            return [start,i]
        }
    }
    start ++        
    }

    
};

twoSum([2,7,11,15], 9)

File: Completed-Algorithms/LeetCode1_Two_Sum/twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        dictionary = {}
        for i, num in enumerate(nums):
            if target - num in dictionary:
                return[dictionary[target - num], i]
            if not num in dictionary:
                dictionary[num] = i
    

        
